src/Http/Session/CacheSession.php

```diff
@@ -20,6 +20,7 @@
 
 use Cake\Cache\Cache;
 use InvalidArgumentException;
+use ReturnTypeWillChange;
```

Comment:
I can't find docs for this, but I'm going to assume PHP8.1 reads this attribute and turns of return type comparisons?

Reply:
It isn't well documented, but this attribute will suppress the deprecation of mismatched tentative return types.

https://wiki.php.net/rfc/internal_method_return_types